src/sidebar/components/tag-list.js

```diff
@@ -32,6 +32,7 @@ function TagList({ annotation, serviceUrl, settings, tags }) {
             <a
```

Comment:
Might be wise to add it to the tag editor's AutocompleteList too. Or even a just blanket the prop on all AutocompleteList components. (we are only using it once of course) Perhaps on the `<div class="autocomplete-list">` tag?

Reply:
Hmm ... it depends on where the suggestions for an autocomplete list are coming from. For the moment I added the `lang=""` attribute just to suggestions rendered by the tag editor when it uses `AutocompleteList`, since we know they are definitely user-supplied (as opposed to coming from some source where we know the language).

Reply:
Yes, thinking about this more, its highly likely the user added all these tags anyway, perhaps we let it be whatever the UI is.